Test/dafny0/ArrayElementInit.dfy

```diff
@@ -73,8 +73,8 @@ method Q0(s: Six, y: int) returns (a: array<Six>)
 {
   if
   case true =>  a := new Six[10](_ => s);
-  case true =>  a := new Six[10](x => 6+x);
-  case true =>  a := new Six[10](_ => y);
+  case true =>  // a := new Six[10](x => 6+x); // type error
```

Comment:
Why are these and many other lines below commented out? Do they no longer produce the same output?

Reply:
I just see two lines commented out because they now produce type resolution errors, which would stop the verification checks of the rest of the file.

Reply:
cleaned up these two test files